centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -291,7 +292,7 @@ object Operations extends StrictLogging {
             case s: TerminalStatus =>
               CentaurCromwellClient.metadata(workflow) flatMap { metadata =>
                 val message = s"Unexpected terminal status $s while waiting for one of [${expectedStatuses.mkString(", ")}] (workflow ID: ${workflow.id})"
-                IO.raiseError(CentaurTestException(message, testDefinition, workflow, metadata))
+                IO.raiseError(CentaurTestException(message, testDefinition, workflow, metadata, previousWorkflows))
```

Comment:
I remember looking into making this change as part of CROM-6807 and thinking that there were a lot of exceptions created in this file that would need to be changed to include the previous workflows. My thinking was that any exception created as part of running the relevant test formats would need to be adjusted, yes/no?

Reply:
Yeah upon further review the strategy I had here would not cut it due to all the places where exceptions could be thrown. I'll see if I can get something more general and less painful to work and if not we might just keep things as they are with restored `retryTestFailures` behavior.